samples/cpp/benchmark_app/main.cpp

```diff
@@ -776,8 +794,8 @@ int main(int argc, char* argv[]) {
 
             if (useGpuMem) {
                 auto outputBlobs = ::gpu::getRemoteOutputBlobs(exeNetwork, inferRequest->getOutputClBuffer());
-                for (auto& output : exeNetwork.GetOutputsInfo()) {
-                    inferRequest->setBlob(output.first, outputBlobs[output.first]);
+                for (auto& output : exeNetwork.outputs()) {
+                    inferRequest->setBlob(output.get_any_name(), outputBlobs[output.get_any_name()]);
```

Comment:
Please use new API

Reply:
please clairfy. It uses outputs already. setTensor (formerly known as setBlob) is our own function, please don't be confused.